docs/source/optimizations/cq/cq_compile_css.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "eb89a144",
```

Comment:
<p>This import should match the formatting of the other optimization notebooks. See: https://superstaq.readthedocs.io/en/latest/optimizations/qscout/qscout_css.html. Ditto for <code>qss</code> notebook </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done</p><p> </p>